release.sh

```diff
@@ -0,0 +1,289 @@
+#!/bin/bash
+
+usage() {
+    cat <<EOF
+Usage: $0 [-h]
+
+This is the release script for Membrane RTC Engine.
+Run \`$0\`, then follow the instructions.
+
+Available options:
+    -h    Print this help and exit
+EOF
+}
+
+## Global variables
+# The order of these repos is important: Engine must be released first, then WebRTC, then the rest
+REPOS="engine webrtc hls rtsp file sip"
```

Comment:
Could we generalize it, so when we add a new endpoint we won't have to remember to add it here?